images/kubevirt-infra-bootstrap/Dockerfile

```diff
@@ -16,3 +16,9 @@ RUN apt-get update \
         python3 \
         python3-pip \
         && rm -rf /var/lib/apt/lists/*
+RUN export KUSTOMIZE_DIR=/opt/kustomize \
```

Comment:
Would https://docs.docker.com/engine/reference/builder/#env help with all the exports?

Reply:
Sorry, it is not actually that helpful since it would be lost after the build is done.